scripts/codegen_driverwrappers/generate_driver_wrappers.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
```

Comment:
Please move this script to the `scripts` directory, and its data files to `scripts/data_files`.